tutor/templates/config.yml

```diff
@@ -71,6 +71,7 @@ PLATFORM_NAME: "My Open edX"
 PLUGINS: []
 REDIS_HOST: "redis"
 REDIS_PORT: 6379
+REDIS_DB: 1
```

Comment:
This DB would only be used for edx-platform, so if we decide to add this extra parameter, I suggest we rename it to `OPENEDX_REDIS_DB`.

Reply:
It might also be used by plugins, such as discovery and ecommerce.

Reply:
Right. I see that the discovery cache is configured (in Tutor) to use DB=1, while ecommerce uses DB=0 for celery tasks; the latter is almost certainly incorrect. I agree with you that we should introduce configuration settings. I propose that we use separate databases for each, with the following names: `OPENEDX_CACHE_REDIS_DB`, `OPENEDX_CELERY_REDIS_DB`, `DISCOVERY_CACHE_REDIS_DB`, `ECOMMERCE_CELERY_REDIS_DB`.

Would you like to make the discovery pull request? Will you update this PR with this proposed change?

Reply:
sure, I will make pr for discovery.